processing/src/main/java/io/druid/segment/ColumnSelectorBitmapIndexSelector.java

```diff
@@ -115,7 +115,56 @@ public BitmapFactory getBitmapFactory()
   public BitmapIndex getBitmapIndex(String dimension)
```

Comment:
Sure, I changed `getBitmapIndex(String dimension, String value)` to return null for that case.

re: reusing logic, I'd prefer to leave this as-is, I assume you mean this block:
```
if (Strings.isNullOrEmpty(value)) {
        return bitmapFactory.complement(bitmapFactory.makeEmptyImmutableBitmap(), getNumRows());
      } else {
        return bitmapFactory.makeEmptyImmutableBitmap();
      }
```

I could have  `public ImmutableBitmap getBitmap(int idx)` in the anonymous BitmapIndex call `getBitmapIndex(String dimension, String value)`, but I would still have to check for idx == 0, and would need to pass in some bogus non-null/empty value for the idx != 0 case. 

I think it looks cleaner as-is.